docs/to_clarify.md

```diff
@@ -0,0 +1,8 @@
+## Question
+1. What are the existing solutions that was mentioned in point 2 of the "Possible Use Cases"
+
+1. What are the digital solutions that the industry have tried? To potentially prevent designing similar solutions 
```

Comment:
We could be more specific: As mentioned under "Context" that "hotels... have tried different digital check-in solutions to limited success", are you able to show us exactly what are some of the solutions that have been tried and in which aspects are they lacking? Having these references would help us to come up with a better solution.